packages/pipeline/src/Counter.tsx

```diff
@@ -0,0 +1,154 @@
+import React, { forwardRef, ReactElement, ReactNode, Ref } from 'react';
+import PropTypes from 'prop-types';
+import { css, cx } from '@leafygreen-ui/emotion';
+
+import {
+  getRootStyle,
+  getChildStyle,
+  getChevronStyle,
+  Size,
+  Variant,
+  layout,
+  colors,
+  counterAttr,
+  counterChevronAttr,
+} from './styles';
+
+interface StateForStyles {
+  size: Size | undefined;
```

Comment:
```suggestion
  size?: Size;
```

Is equivalent to what you have here. Generally preferable to use the question mark suffix. Ditto for variant field.

Reply:
Resolved by 275f61c.